stylist/admin.py

```diff
@@ -13,7 +14,10 @@ class StyleAdmin(admin.ModelAdmin):
     list_display = ('id', 'name', 'slug', 'site', 'attrs', 'enabled')
     list_filter = ('site', 'enabled')
     search_fields = ('name', 'slug')
-    actions = [compile_styles]
-
+    
+    def __init__(self, model=Style, admin_site=admin.site) -> None:
+        super().__init__(model, admin_site)
+        if not getattr(settings, 'STYLIST_IGNORE_SASS', False):
```

Comment:
the readme changes are good, but i meant more that this type of getattr check should default to True instead of False when we're checking for the setting; that way the user doesn't have to worry about editing it in the settings file unless they're setting `STYLIST_IGNORE_SASS=False`

Reply:
aaah okay okay